### PR TITLE
spread: Use /home/gopath in spread.yaml

### PR DESCRIPTION
**Pull request by mvo5**
The latest snap-confine does not allow to be in a path that is no
longer valid once inside the snap-confine. This breaks because
snap-confine only mounts a subset of the directories of the host "/"
into the snap chroot (unlike before when ubuntu-core-launcher left
"/" alone).

### CODE REVIEW

File: spread.yaml [hunk 1]
```diff
@@ -3,8 +3,8 @@ project: snapd
 environment:
     REUSE_PROJECT: $(echo $REUSE_PROJECT)
     PROJECT_PATH: /gopath/src/github.com/snapcore/snapd
```
Comment (fgimenez) [inline spread.yaml]: Thanks for this @mvo5 :) Don't we need to modify `PROJECT_PATH` too?

Comment (fgimenez) [inline spread.yaml]: We also need to use `$PROJECT_PATH` in https://github.com/snapcore/snapd/blob/master/tests/main/gccgo/task.yaml#L13

Thanks!

**Comment by zyga** [comment]
Looks good now :)

File: snap/implicit_test.go [hunk 2]
```diff
@@ -42,7 +42,7 @@ func (s *InfoSnapYamlTestSuite) TestAddImplicitSlotsOutsideClassic(c *C) {
 	c.Assert(info.Slots["network"].Interface, Equals, "network")
 	c.Assert(info.Slots["network"].Name, Equals, "network")
 	c.Assert(info.Slots["network"].Snap, Equals, info)
-	c.Assert(info.Slots, HasLen, 19)
+	c.Assert(info.Slots, HasLen, 20)
```
Comment (zyga) [inline snap/implicit_test.go]: Uh?

Comment (mvo5) [inline snap/implicit_test.go]: Yes, master broken. I can do a separate branch for this.

File: spread.yaml (hunk 1 above)
Comment (mvo5) [inline spread.yaml]: Yes, silly me!

File: snap/implicit_test.go (hunk 2 above)
Comment (zyga) [inline snap/implicit_test.go]: ! Ok thanks for explaining this. No need to have a separate branch

**Comment by niemeyer** [comment]
Single test broken which is a real unrelated bug:

    - linode:ubuntu-16.04-64-grub:tests/main/writable-areas

**Pull request merged**